src/libraries/Microsoft.PowerFx.Core/Public/Values/TableValue.cs

```diff
@@ -190,5 +193,36 @@ public override void Visit(IValueVisitor visitor)
         {
             visitor.Visit(this);
         }
+
+        public override string ToExpression()
+        {
+            if (!Rows.Any())
+            {                
```

Comment:
suggested comment:

// Table() is not legal, so we need an alternate expression to capture the table's type